lib/streamlit/elements/widgets/button.py

```diff
@@ -308,6 +312,15 @@ def download_button(
 
         """
         ctx = get_script_run_ctx()
+
+        # Checks whether the entered button type is one of the allowed options - either
+        # "primary" or "secondary"
```

Comment:
nit: I feel like this comment isn't really necessary as the code itself is pretty self-explanatory, especially with the docstring above it that also describes what the parameter does